internal/signalfx-agent/.gitignore

```diff
@@ -1 +1,4 @@
 pkg/monitors/zcodegen/monitorcodegen
+
+# Coverage
+coverage/*
```

Comment:
Was this one for tests or is it really needed? My vague memory tells me that the one at the top level is enough, no?

Reply:
I tested, it's not necessary. Removed. 👍